host-configs/llnl/toss_4_x86_64_ib/clang@14.0.6.cmake

```diff
@@ -0,0 +1,56 @@
+# Copyright (c) 2017-2023, Lawrence Livermore National Security, LLC and
+# other BLT Project Developers. See the top-level LICENSE file for details
+# 
+# SPDX-License-Identifier: (BSD-3-Clause)
+
+#------------------------------------------------------------------------------
+# Example host-config file for the ruby cluster at LLNL
```

Comment:
Is this for the ruby cluster? 

Might be better to just say something like "a cluster on the toss4 platform" ?

Reply:
Thanks -- I don't think there's anything specific to quartz, so the latter is probably better.